packages/swapper/src/swappers/ThorchainSwapper/evm/utils/getThorTxData.ts

```diff
@@ -33,8 +34,8 @@ export const getThorTxInfo = async ({
   if (maybeInboundAddress.isErr()) throw maybeInboundAddress.unwrapErr()
   const inboundAddress = maybeInboundAddress.unwrap()
 
-  const router = inboundAddress.router
-  const vault = inboundAddress.address
+  const router = getAddress(inboundAddress.router as string)
+  const vault = getAddress(inboundAddress.address as string)
```

Comment:
Cast is unneeded, `address` _is_ of type `string` already.
 
```suggestion
  const vault = getAddress(inboundAddress.address)
```